provisioning/service/src/provisioningserviceclient.ts

```diff
@@ -221,6 +221,7 @@ export class ProvisioningServiceClient {
   }
 
   private _createOrUpdate(endpointPrefix: string, enrollment: any, callback?: (err: Error, enrollmentResponse?: any, response?: any) => void): void {
+
     /*Codes_SRS_NODE_PROVISIONING_SERVICE_CLIENT_06_009: [The `createOrUpdateIndividualEnrollment` method shall throw `ReferenceError` if the `IndividualEnrollment` argument is falsy.]*/
```

Comment:
nit: unnecessary blank line

Reply:
Fixed.

---
In reply to: [154166412](https://github.com/Azure/azure-iot-sdk-node/pull/165#discussion_r154166412) [](ancestors = 154166412)